components/layout_thread/lib.rs

```diff
@@ -1740,11 +1741,11 @@ fn get_root_flow_background_color(flow: &mut Flow) -> webrender_api::ColorF {
 fn get_ua_stylesheets() -> Result<UserAgentStylesheets, &'static str> {
     fn parse_ua_stylesheet(
         shared_lock: &SharedRwLock,
-        filename: &'static str,
+        filename: &str,
+        content: String,
```

Comment:
This could take `Vec<u8>` or `&[u8]` instead, and then the callers could use `read_bytes` instead of `read_string` to avoid needless UTF-8 validation.